docker/h2-proxy/default.conf

```diff
@@ -0,0 +1,12 @@
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    ssl_certificate /etc/nginx/ssl/server.crt;
+    ssl_certificate_key /etc/nginx/ssl/server.key;
+    ssl_session_cache  builtin:1000  shared:SSL:10m;
```

Comment:
I wouldn't change the default value (as this container is only intended for dev, we can keep the config minimal).